CoffeeChat/Controller/HomeViewController.swift

```diff
@@ -6,25 +6,37 @@
 //  Copyright © 2020 cuappdev. All rights reserved.
 //
 
+import GoogleSignIn
 import UIKit
 
 class HomeViewController: UIViewController {
 
+    private let logoutButton = UIButton()
+
     override func viewDidLoad() {
         super.viewDidLoad()
+        view.backgroundColor = .white
+
+        logoutButton.setTitle("Log Out", for: .normal)
+        logoutButton.setTitleColor(.black, for: .normal)
+        logoutButton.addTarget(self, action: #selector(logoutPressed), for: .touchUpInside)
+        view.addSubview(logoutButton)
 
-        // Do any additional setup after loading the view.
+        setupConstraints()
```

Comment:
super minor, but since the home view currently is one button, it could be simpler to put the contents of `setupConstraints` in the `viewDidLoad`

Reply:
i agree with this, lets just move the constraints into the `viewDidLoad`